Wikipedia/UI-V5/WMFTitleInsetRespectingButton.m

```diff
@@ -0,0 +1,18 @@
+
+#import "WMFTitleInsetRespectingButton.h"
+
+@implementation WMFTitleInsetRespectingButton
+
+/**
+ *  UIButton does not take into acocunt its insets when calculating intrinisic content size
+ *  http://stackoverflow.com/a/17806333/48311
+ *
+ */
+- (CGSize)intrinsicContentSize {
+    CGSize s = [super intrinsicContentSize];
+
+    return CGSizeMake(s.width + self.titleEdgeInsets.left + self.titleEdgeInsets.right,
```

Comment:
Crash seems unrelated.

as far as I know, `[super intrinsicConstentSize]` always returns a value as does `self.titleEdgeInsets` Do you know a situation where those would return garbage?